Common/OpenCL/ITKimprovements/itkOpenCLUtil.h

```diff
@@ -40,6 +40,10 @@
 #include <sstream>
 #include <typeinfo>
 
+#ifndef CL_TARGET_OPENCL_VERSION
+#  define CL_TARGET_OPENCL_VERSION 120
```

Comment:
This particular header file (itkOpenCLUtil.h) does not have any OpenCL `#include`, so here `#define CL_TARGET_OPENCL_VERSION` does not seem necessary, does it?

If you still think both itkOpenCL.h and itkOpenCLUtil.h should do `#define CL_TARGET_OPENCL_VERSION`, I hope that the code redundancy can be avoided.

Reply:
@N-Dekker yes, I can remove it form *itkOpenCLUtil.h*. It was placed there because [*opencl.h* is include'd in this file in ITK](https://github.com/InsightSoftwareConsortium/ITK/blob/7e52961776e242290f9725f273b3e3c8f16b8cf8/Modules/Core/GPUCommon/include/itkOpenCLUtil.h#L34-L42). Eventually, we should push the improvements in elastix (which are great) upstream to avoid duplicated effort.